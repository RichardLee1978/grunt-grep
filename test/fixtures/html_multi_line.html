<!doctype html>
<html lang="en">
<head>
	<!-- new:s -->
	<meta charset="UTF-8">
	<title>Document</title>
	<!-- new:e -->
</head>
<body>
	<div id='home'>
	<div id='away'>
	<span></span>
	<!-- old:s -->
	<img src="./asd" alt="">
	<!-- old:e -->
</body>
</html>