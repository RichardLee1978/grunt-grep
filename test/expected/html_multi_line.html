<!doctype html>
<html lang="en">
<head>
	<!-- new:s -->
	<meta charset="UTF-8">
	<title>Document</title>
	<!-- new:e -->
</head>
<body>
	<div id='home'>
	<div id='away'>
	<span></span>
</body>
</html>